packages/vuetify/src/components/VCalendar/VCalendarDaily.ts

```diff
@@ -65,14 +65,15 @@ export default CalendarWithIntervals.extend({
       ])
     },
     genHeadIntervals (): VNode {
+      const slot = this.$scopedSlots['interval-header']
       const width: string | undefined = convertToUnit(this.intervalWidth)
 
       return this.$createElement('div', {
         staticClass: 'v-calendar-daily__intervals-head',
         style: {
           width,
         },
-      })
+      }, slot ? slot({}) : undefined)
```

Comment:
`getSlot(this, 'interval-header')`

Reply:
Yeah I'll switch to that. That function would be a little more useful if `data?: object` was `getData?: () => object` so if I have slot data it is only created if a slot is actually returned, as opposed to needing to be created and potentially never used.

Reply:
Feel free to add it, `if (typeof data === 'function') data = data()`

Reply:
or `data instanceof Function`, that seems to be the best option https://stackoverflow.com/questions/5999998/check-if-a-variable-is-of-function-type#answer-19717946